src/frameworks/web/components/organisms/Header/Header.tsx

```diff
@@ -37,88 +41,102 @@ const HeaderButtonContainer = styled.div`
 const HeaderSidebarButton = styled.img`
   float: right;
   cursor: pointer;
+
+  width: ${(props: IHeaderItem) => (props.size === 'xs' ? '20px' : '40px')};
+  height: ${(props: IHeaderItem) => (props.size === 'xs' ? '20px' : '40px')};
+  margin: ${(props: IHeaderItem) => (props.size === 'xs' ? '14px 0' : '20px 0')}px;
```

Comment:
```suggestion
  margin: ${(props: IHeaderItem) => (props.size === 'xs' ? '14px' : '20px')} 0px;
```

이렇게 작성하는 것이 더욱 가독성 있어보이는 것 같습니다.